core/src/main/resources/org/jobrunr/dashboard/frontend/src/components/recurring-jobs/recurring-jobs.js

```diff
@@ -175,7 +175,7 @@ const RecurringJobs = (props) => {
                                                         {recurringJob.jobName}
                                                     </TableCell>
                                                     <TableCell>
-                                                        {cronstrue.toString(recurringJob.cronExpression)}
+                                                        {cronstrue.toString(recurringJob.scheduleExpression)}
```

Comment:
This will fail for durations. We could check if the scheduleExpression starts with 'P' and depending on that show a different value.